src/test/scala/com/dwolla/rabbitmq/policies/JsonCodecSpec.scala

```diff
@@ -0,0 +1,42 @@
+package com.dwolla.rabbitmq.policies
+
+import com.eed3si9n.expecty.Expecty.expect
+import io.circe._
+import io.circe.literal._
+import io.circe.syntax._
+import munit.ScalaCheckSuite
+import org.scalacheck.Prop
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.{Arbitrary, Gen}
+
+
+class JsonCodecSpec extends ScalaCheckSuite {
+
+  private val genPolicyDefinition: Gen[PolicyDefinition] = {
```

Comment:
This block is duplicated in `HandlerSpec`, so I think it could be removed from both classes and consolidated in the `Arbitraries` trait. In order to do that, it can no longer be `private`:

```scala
val genPolicyDefinition: Gen[PolicyDefinition] =
  for {
    haMode <- arbitrary[String]
    haParams <- arbitrary[Int]
    messageTtl <- arbitrary[Option[Int]]
  } yield PolicyDefinition(haMode, haParams, messageTtl)
implicit val arbPolicyDefinition: Arbitrary[PolicyDefinition] = Arbitrary(genPolicyDefinition)
```

(I also cleaned the example up by renaming the field names to make them more idiomatic, and also removed the unnecessary `{}` around the for-comprehension, which are typically omitted in Scala unless they're actually required semantically.)

Once that's added to `Arbitraries`, then this class will also need to extend `Arbitraries` to bring the implicit into scope:

```scala
class JsonCodecSpec
  extends ScalaCheckSuite
    with Arbitraries {
```